functions/src/api/donations/donations.ts

```diff
@@ -15,4 +15,33 @@ router.post('/', async (req: any, res: any) => {
     res.json(donation);
 });
 
+router.get('/:id', async (req: any, res: any) => {
+    const donationId = req.params.id;
+    const donation = await get(DONATIONS_COLLECTION, donationId);
+    const lineItemIds = donation.line_item_ids;
+
+    donation.line_items = [];
+
+    lineItemIds.array.forEach(async (lineItemId:any) => {
+        donation.line_items[donation.line_items.length] = await getByItemId(lineItemId)
+    });
+
+    res.status(201);
```

Comment:
201 is Created. You want to use 200 which I believe is the default so you can lose this line and the same for the get all